### DISCUSSION TITLE
Update gitlab.tpl file schema

### DISCUSSION THREAD

**Discussion by dioguerra** [aquasecurity/trivy#5119] [Bugs]
### Description

Using the contrib/gitlab.tpl shows in gitlab that the template is outdated:

```
[Schema] version 14.0.6 for report type container_scanning is deprecated. However, GitLab will still attempt to parse and ingest this report. Upgrade the security report to one of the following versions: 15.0.0, 15.0.1, 15.0.2, 15.0.4.
```

Which kinda works, but should be updated

### Desired Behavior

No warning

### Actual Behavior

Warning

### Reproduction Steps

```bash
gitlab CI:


wget https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/gitlab.tpl
        wget --output-document - https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin ${TRIVY_VERSION:-latest}
        trivy image --exit-code 0 --cache-dir .trivycache/ --no-progress --format template --template "@gitlab.tpl" -o gl-container-scanning-report.json ${TRIVY_INPUT}
```

### Target

None

### Scanner

None

### Output Format

Template

### Mode

Standalone

### Debug Output

```bash
No need
```

### Operating System

docker:20.10.17

### Version

```bash
latest as of today
```

### Checklist

- [ ] Run `trivy image --reset`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment]
Track https://github.com/aquasecurity/trivy/pull/4035
